test-utils/state-viewer/src/cli.rs

```diff
@@ -279,3 +285,16 @@ impl EpochInfoCmd {
         );
     }
 }
+
+#[derive(Clap)]
+pub struct RocksDBStatsCmd {
+    /// Location of the dumped Rocks DB stats.
+    #[clap(long, parse(from_os_str))]
+    file: Option<PathBuf>,
+}
+
+impl RocksDBStatsCmd {
+    pub fn run(self, home_dir: &Path) {
+        assert_matches!(get_rocksdb_stats(home_dir, self.file), Ok(_));
```

Comment:
I don't think its worth pulling in `assert_matches!` as a runtime dependency just for this.